current_version.json

```diff
@@ -1,13 +1,13 @@
 {
-    "tag_name": "v1.0.2",
+    "tag_name": "v1.0.5",
     "assets": [
-        {
-            "browser_download_url": "https://github.com/FriendsOfGalaxy/galaxy-integration-steam/releases/download/v1.0.2/windows.zip",
-            "name": "windows.zip"
-        },
-        {
-            "browser_download_url": "https://github.com/FriendsOfGalaxy/galaxy-integration-steam/releases/download/v1.0.2/macos.zip",
-            "name": "macos.zip"
-        }
+		{
+			"browser_download_url": "https://github.com/ABaumher/galaxy-integration-steam/releases/download/Beta/steam_v1.0.5.zip",
+			"name": "steam_v1.0.5.zip"
+		},
+		{
+			"browser_download_url": "https://github.com/ABaumher/galaxy-integration-steam/releases/download/Beta/steam_v1.0.4_mac.zip",
```

Comment:
Typo? steam_v1.0.4_mac.zip -> steam_v1.0.5_mac.zip

Reply:
Good catch. Right now that flew didn't do anything but once we fix the gog integration that value needs updating